kv/local_sender_test.go

```diff
@@ -43,6 +44,36 @@ func TestLocalSenderAddStore(t *testing.T) {
 	}
 }
 
+func TestLocalSenderRemoveStore(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	ls := NewLocalSender()
+
+	store := storage.Store{}
```

Comment:
The requirement really comes from `store.Start` - it's technically invalid to create a store with `Store{}`; you need to use `NewStore` and `Store.Start` to do anything real. But that's a lot of machinery to introduce for this test. `NewStore` creates a store with ID 0 and then it changes to the real id in `Start()`. We should probably check in `AddStore` that the ID is non-zero, since that indicates a programming error (adding a non-started store whose ID will change when it is started).